src/industry_cmd.cpp

```diff
@@ -1826,7 +1852,7 @@ static void DoCreateNewIndustry(Industry *i, TileIndex tile, IndustryType type,
 		}
 
 		for (auto &p : i->produced) {
-			p.history[LAST_MONTH].production += p.rate * 8;
+			p.history[LAST_MONTH].production += ScaleByCargoScale(p.rate * 8, false);
 		}
```

Comment:
Not actually this line. `ChopLumberMillTrees()` appears to be unaffected by any scaling as it's still executed on the fixed tick interval, and the production algorithm is not changed. The constant `45` should be scaled?

Reply:
Yes, good catch. I agree it's best to scale the produced amount, while leaving the rate of tree-cutting alone.